packages/runtime/test/src/logicsig.ts

```diff
@@ -1,27 +1,66 @@
 import { decodeAddress, multisigAddress, MultisigMetadata } from "algosdk";
 import { assert } from "chai";
-
+import { convert } from "@algo-builder/algob";
 import { AccountStore } from "../../src/account";
 import { RUNTIME_ERRORS } from "../../src/errors/errors-list";
 import { Runtime } from "../../src/runtime";
 import { useFixture } from "../helpers/integration";
 import { expectRuntimeError } from "../helpers/runtime-errors";
+import { types } from "@algo-builder/web";
 
 const programName = "escrow.teal";
 const multiSigProg = "sample-asc.teal";
+const crowdFundEscrow = "crowdFundEscrow.teal";
 
-describe("Logic Signature", () => {
+describe("Load logic for teal program", () => {
```

Comment:
I would prefer change back to `Logic Signature` since many test case here use logic signature

Reply:
done